sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -71,27 +68,29 @@ class StreamExecution(
 
   import org.apache.spark.sql.streaming.StreamingQueryListener._
 
-  private val pollingDelayMs = sparkSession.sessionState.conf.streamingPollingDelay
+  protected val pollingDelayMs: Long = sparkSession.sessionState.conf.streamingPollingDelay
 
-  private val minBatchesToRetain = sparkSession.sessionState.conf.minBatchesToRetain
+  protected val minBatchesToRetain: Int = sparkSession.sessionState.conf.minBatchesToRetain
```

Comment:
since this is a base class for both microbatch and continuous processing, is it right to put this variable here?

Reply:
yes, tweaking the var names may make it look better